### PR TITLE
[Doc Improvement][Mobile Bot Support Tag Mention][3470053]

### PR DESCRIPTION
**Pull request by v-kushals**
Updated the following pages announcing that tag mentions by bots are now supported in web and desktop clients:
* [Channel and group chat conversations with a bot](https://learn.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/channel-and-group-conversations?tabs=dotnet)
* [What's new for developers in Microsoft Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/whats-new?pivots=ga-feature)
* [Microsoft Teams developer documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/mstdd-landing)

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5769a0f2dcaac79907f76b1bc211ae8700ea8b50/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/737ed015-a673-48de-b0f4-d1c34495191f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/5769a0f2dcaac79907f76b1bc211ae8700ea8b50/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1b8391c-89e9-438b-b241-87e3d564e407) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5769a0f2dcaac79907f76b1bc211ae8700ea8b50/msteams-platform%2Fwhats-new.md) | 94 | 53 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5bb3d42-e0c2-4438-838d-cfd48fe6512a) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/08dc649af97ff6dbcb4b34552d34a3249c481095/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cf119452-b1f7-4b2f-aac5-f14dfd80457f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/08dc649af97ff6dbcb4b34552d34a3249c481095/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/efb5a131-1428-4ebd-8c8a-cb908c223e1f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/08dc649af97ff6dbcb4b34552d34a3249c481095/msteams-platform%2Fwhats-new.md) | 94 | 53 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bb563372-2089-41d8-8d8f-13302cf31b0c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/76b8405a81c3929b09c1432c521996172ce410ce/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d6e66113-69a3-4288-8a3d-e9c90b5e729c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/76b8405a81c3929b09c1432c521996172ce410ce/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/42ccefc8-c88a-4787-8586-917871b1f107) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/8d8285355cc3e3a618874a64301e4a37ea69ac2c/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9fbadab2-9f4f-4c37-a971-bbdcee54e441) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/8d8285355cc3e3a618874a64301e4a37ea69ac2c/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/638d2ad2-5c92-471d-a666-b1348aede728) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by limamicro — DISMISSED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/026e0e5c1372709b5c5c03405aaa884edb1b2856/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/430d1559-ed34-4196-82ec-7952b9b7b462) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b71a09b299e094c482ee353cddc0c559a568b6a3/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c43fb1bf-7b8b-4dc1-8337-f56ba0c48a4f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b71a09b299e094c482ee353cddc0c559a568b6a3/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/50894028-4233-4d05-bbf7-922f31912322) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b71a09b299e094c482ee353cddc0c559a568b6a3/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8830dc3c-7596-468d-9c40-f8ffebe85eca) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/651d0e3f3105fa18e7bb7c14bd5855ce7f329297/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f2e7fd3b-730e-4e9d-8141-9d33d566ba5b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/651d0e3f3105fa18e7bb7c14bd5855ce7f329297/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3cd879dd-0c55-4d42-b9f8-80431442ff7a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/651d0e3f3105fa18e7bb7c14bd5855ce7f329297/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7d94034b-c8de-4b4a-ac7b-6dfe2d16de29) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/968feb1326d57edb7ef08645e261ac657433fca0/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c5b66d0c-e059-4107-8b0e-23ba173888bb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/968feb1326d57edb7ef08645e261ac657433fca0/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fd036739-d71b-4742-bee7-34a09f66d351) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/968feb1326d57edb7ef08645e261ac657433fca0/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6e6ff09-b459-4a40-a9df-8e11dcfa9f10) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f0795cecbbc921568b650b4084e38a5526331f44/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/041b8d6d-0099-4ccb-831b-151858ee549d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/f0795cecbbc921568b650b4084e38a5526331f44/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0dfe3af-62da-4fa9-8ccb-d859fb80da23) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f0795cecbbc921568b650b4084e38a5526331f44/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2787c66a-7b34-44ec-832b-13e68230f7a9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/db34d8ab572e4bd8982851dbf14b554814ae44f1/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8932155a-68b4-4b79-beed-d2230d7465af) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/db34d8ab572e4bd8982851dbf14b554814ae44f1/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eff5da45-7a5e-41fc-95cc-1ce8108b404f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/db34d8ab572e4bd8982851dbf14b554814ae44f1/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/69893d6c-fde7-4b2c-ba62-4453f5c24013) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by limamicro — DISMISSED** [review]

File: msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md [hunk 1]
```diff
@@ -303,8 +303,7 @@ Your bot can mention tags in text messages and Adaptive Cards posted in channels
 
 > [!NOTE]
 >
-> * Tag mentions are available only in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md).
-> * Tag mentions are supported in Teams desktop and web clients. However, it's not supported in Teams mobile client.
+> * Tag mentions are supported in Teams desktop and web clients only.
```
[the remark this answers is not in the document]
Comment (v-kushals) [reply msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md]: Updated

File: msteams-platform/mstdd-landing.yml [hunk 2]
```diff
@@ -44,10 +44,12 @@ landingContent:
 #      - text: Adaptive Cards overflow menu
 #        url: ~/task-modules-and-cards/cards/cards-format.md
 
-- title: Whats new
+- title: What's new
   linkLists:
     - linkListType: whats-new
       links:
+      - text: Bots can mention tags in channels in Teams web and desktop clients
```
[the remark this answers is not in the document]
Comment (v-kushals) [reply msteams-platform/mstdd-landing.yml]: Updated

File: msteams-platform/whats-new.md [hunk 3]
```diff
@@ -26,8 +26,9 @@ Teams platform features that are available to all app developers.
 
 **2023 December**
 
-**December 12, 2023**: [Use Teams AI library to build apps that can leverage LLMs to facilitate more natural conversational interactions with users, guiding that conversation into your apps skills](bots/how-to/Teams%20conversational%20AI/teams-conversation-ai-overview.md).
+**December 14, 2023**: [Bots can mention tags in text messages and Adaptive Cards posted in channels in Teams web and desktop clients](bots/how-to/conversations/channel-and-group-conversations.md#tag-mention).
```
[the remark this answers is not in the document]
Comment (v-kushals) [reply msteams-platform/whats-new.md]: Updated

**Review by v-habibaumme — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/24ac42c597b6a68425bebf5c2f6ed613aeea40ab/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2c6f045f-0609-4b38-841f-0072cc58a457) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/24ac42c597b6a68425bebf5c2f6ed613aeea40ab/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e1f4609-7de7-4beb-909b-27f2b82c9c77) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/24ac42c597b6a68425bebf5c2f6ed613aeea40ab/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8427f8f8-8fc5-45cc-87ab-5e302132e7fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dcf5ef8250334567d9e32ae0120f492cef020f7e/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 93 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d7543eeb-da30-4ae5-a84f-b3747388c1e9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/dcf5ef8250334567d9e32ae0120f492cef020f7e/msteams-platform%2Fmstdd-landing.yml) | 94 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bc73d428-4b1e-4ee2-852c-9d58bd686223) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dcf5ef8250334567d9e32ae0120f492cef020f7e/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ddba6264-5ff7-49ac-b0e2-85cf1ca8d866) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — APPROVED** [review]
Approved

**Pull request merged**